test/harness/e2e/vm/domain-template.xml

```diff
@@ -26,7 +34,7 @@
       </target>
       <alias name='serial0'/>
     </serial>
-    <console type='pty' tty='/dev/pts/2'>
+    <console type='pty'>
```

Comment:
fixing typo, this shouldn't be specific about the pts... libvirt (or qemu) will do auto-allocation